tools/ci-build.sh

```diff
@@ -94,17 +92,18 @@ P4C_PIP_PACKAGES="ipaddr \
                   scapy==2.4.5 \
                   clang-format>=15.0.4"
 
-
 if [[ "${DISTRIB_RELEASE}" == "18.04" ]] ; then
```

Comment:
```Suggestion
if [[ "${DISTRIB_RELEASE}" == "18.04" ]] || [[ "$(which simple_switch 2> /dev/null)" != "" ]] ; then
```
With this change, the `p4lang-bmv2` package would be installed only if BMv2 is not already installed.
For  example, BMv2 should already be available in the `p4lang/behavioral-model:latest` container image.